src/main/java/seedu/address/model/task/Entry.java

```diff
@@ -184,4 +188,39 @@ public final String tagsString() {
             return buffer.substring(0, buffer.length() - DELIMITER.length());
         }
     }
+    
+    //@@author A0121501E
+    public int compareTo(Entry o) {
```

Comment:
Don't implement the compareTo function. It should be implemented by the class that extends Entry.